src/components/articles/article/articlePage.js

```diff
@@ -59,13 +60,14 @@ const ArticlePage = () => {
     }
     return (
       <ArticleFrame {...article} />
+
     )
   }
 
   return (
     <div className="article_page_container">
       <BackwardButton url="/articles" _class="article_back_btn" />
-      {displayArticles()}
+      <Content/>
```

Comment:
This component doesn't add a lot of value. Probably better to just render `ErrorPage` and `ArticleFrame` directly.